src/Template/Layout/js/config/locales.js

```diff
@@ -0,0 +1,11 @@
+
+// define locales available to webpack check config/app.php
+// TO-DO dynamic
+const locales = [
+    'en',
+    'it',
+];
+
```

Comment:
Locales should be `I18n.locales` set in `config/app.php`, somehow

Reply:
at the moment is the only way to tell moment and flatpikr which locale to use
one solution could be to use a general conf in .env to be read both in webpack and app.default.php